.github/workflows/build-pkg.yaml

```diff
@@ -0,0 +1,78 @@
+name: build-pkg
+on:
+  workflow_call:
+    inputs:
```

Comment:
Is it possible to specify which values are acceptable for each input type?

Reply:
I don't think so, according to [the docs](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs), [this blog post](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/), and [the source code](https://github.com/actions/languageservices/blob/5dbaa884dbd889a68eb184eb0f28c0f6618264ee/workflow-parser/src/model/workflow-template.ts#L167-L180).

Reply:
this [options](https://github.com/actions/languageservices/blob/5dbaa884dbd889a68eb184eb0f28c0f6618264ee/workflow-parser/src/model/workflow-template.ts#L179) field?

Reply:
It is always good to add some input validation when creating templates. Not a blocker for this PR though. We can look into it later.

Reply:
I think that only works for manually triggered workflows, to give a drop down as seen in [this blog](https://tkolber.medium.com/dropdown-input-choice-workflows-in-github-actions-e95cf140bf61). Could be wrong, though.